packages/cardano-services/src/cli.ts

```diff
@@ -181,6 +214,71 @@ withCommonOptions(
     });
   });
 
+withCommonOptions(withPostgresOptions(program.command('start-blockfrost').description('Start the Blockfros worker')), {
+  apiUrl: BLOCKFROST_WORKER_API_URL_DEFAULT
+})
+  .addOption(
+    new Option('--blockfrost-api-key <blockfrostApiKey>', BlockfrostWorkerOptionDescriptions.BlockfrostApiKey).env(
+      'BLOCKFROST_API_KEY'
+    )
+  )
+  .addOption(
+    new Option('--cache-ttl <cacheTtl>', BlockfrostWorkerOptionDescriptions.CacheTTL)
+      .env('CACHE_TTL')
+      .default(CACHE_TTL_DEFAULT)
+      .argParser((interval) => Number.parseInt(interval, 10))
+  )
+  .addOption(
+    new Option('--create-schema <true/false>', BlockfrostWorkerOptionDescriptions.CreateSchema)
+      .env('CREATE_SCHEMA')
+      .default(CREATE_SCHEMA_DEFAULT)
+      .argParser((createSchema) =>
+        stringOptionToBoolean(createSchema, Programs.BlockfrostWorker, BlockfrostWorkerOptionDescriptions.CreateSchema)
+      )
+  )
+  .addOption(
+    new Option('--drop-schema <true/false>', BlockfrostWorkerOptionDescriptions.DropSchema)
+      .env('DROP_SCHEMA')
+      .default(DROP_SCHEMA_DEFAULT)
+      .argParser((dropSchema) =>
+        stringOptionToBoolean(dropSchema, Programs.BlockfrostWorker, BlockfrostWorkerOptionDescriptions.DropSchema)
+      )
+  )
+  .addOption(
+    new Option('--dry-run <true/false>', BlockfrostWorkerOptionDescriptions.DryRun)
+      .env('DRY_RUN')
+      .default(DRY_RUN_DEFAULT)
+      .argParser((dryRun) =>
+        stringOptionToBoolean(dryRun, Programs.BlockfrostWorker, BlockfrostWorkerOptionDescriptions.DryRun)
+      )
+  )
+  .addOption(
+    new Option('--network <network>', BlockfrostWorkerOptionDescriptions.Network)
+      .env('NETWORK')
+      .argParser((network) => {
+        if (availableNetworks.includes(network as AvailableNetworks)) return network;
+
+        throw new Error(`Unknown network: ${network}`);
+      })
+  )
+  .addOption(
+    new Option('--scan-interval <scanInterval>', BlockfrostWorkerOptionDescriptions.ScanInterval)
+      .env('SCAN_INTERVAL')
+      .default(SCAN_INTERVAL_DEFAULT)
+      .argParser((interval) => Number.parseInt(interval, 10))
+  )
+  .action(async (args: { apiUrl: URL } & BlockfrostWorkerArgs) => {
+    process.stdout.write('Blockfrost worker\n');
+    const blockfrostWorker = await loadAndStartBlockfrostWorker({
```

Comment:
Here you can start the HTTP server, just like `providerServer`

Reply:
Wow!